src/ts/contentScripts/entry/index.ts

```diff
@@ -5,4 +5,53 @@
 
 import '../../features/features'
 import '../../features/dispatcher'
-import '../../features/fuzzy_date'
\ No newline at end of file
+import '../../features/fuzzy_date'
+
+//detect when roam is loaded
+const target = document.getElementById('app')
+if (target) {
+    // Wait for Roam to load before initializing
+    const observer = new MutationObserver(function (mutations) {
+        mutations.forEach(function (mutation) {
+            const newNodes = mutation.addedNodes // DOM NodeList
```

Comment:
um just use type instead of a comment?

Reply:
Before doing all of the work to convert it to typescript I wanted to get this in to get feedback on if this was going to work for you at all. I will clean up things like this before submitting the final PR.